### PR TITLE
Makes gorillas more interesting

### PR DESCRIPTION
**Pull request by Tacolizard**
:cl: Tacolizard
add: Gorillas have a few new emotes when selecting targets
add: Gorillas will now hurl mobs at other mobs outside of their faction
/:cl:

ooga

kinda wip

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm [hunk 1]
```diff
@@ -20,7 +20,7 @@
 	speed = 1
 	melee_damage_lower = 15
 	melee_damage_upper = 18
-	damage_coeff = list(BRUTE = 1, BURN = 1.5, TOX = 1.5, CLONE = 0, STAMINA = 0, OXY = 1.5)
+	damage_coeff = list(BRUTE = 0.5, BURN = 1.5, TOX = 1.5, CLONE = 0, STAMINA = 0, OXY = 1)
```
Comment (Tacolizard) [inline code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm]: yes this is a stealth buff

Comment (Tacolizard) [inline code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm]: yes this is a stealth buff

Comment (Tacolizard) [reply code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm]: reasoning: gorillas 220hp is easily decimated by any kind of lynch mob attack, totally negating the scariness of having a gorilla rampaging on the station. i think this is balanced by the new lower odds of getting a gorilla.

also  if you've ever seen planet of the apes you know they can take a lot of bullets and i wanted to replicate that feeling of a gorilla eating entire magazines of bullets. note that lasers are 1.5 times more effective than base

File: code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm [hunk 2]
```diff
@@ -36,6 +36,16 @@
 	var/list/gorilla_overlays[GORILLA_TOTAL_LAYERS]
 	var/oogas = 0
 
+	search_objects = 1
+	wanted_objects = list(/obj/item/reagent_containers/food/snacks/grown/banana)
```
Comment (Tacolizard) [inline code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm]: if someone else wants to suggest some items a gorilla would want, please comment them here

**Comment by ExcessiveUseOfCobblestone** [comment]
Changelogs should be like PR titles where they explain what was changed effectively without memes. 

Also the idea is to allow them to become a thing a player can inhabit so you may want to keep that in mind when you lower the most common form of damage

File: code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm (hunk 1 above)
Comment (ma44) [reply code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm]: Then everyone would just use welders

**Comment by pubby** [comment]
As chief gorilla of the ape gang, I applaud improvements to gorilla AI.

But don't buff their armor that's just sabotaging your PR.

File: code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm [hunk 3]
```diff
@@ -64,8 +74,23 @@
...
 		if(prob(80))
-			var/atom/throw_target = get_edge_target_turf(L, dir)
-			L.throw_at(throw_target, rand(1,2), 7, src) 
+			var/targetviewers = list()
+			var/atom/throw_target
+			var/targettingviewer
+			//the following code attempts to find a non-jungle mob within sight range of the gorilla
+			//if it finds one or more, it picks one to throw its current attack target at
+			for(var/mob/living/M in oviewers(7, src))
+				if(!("jungle" in M.faction) && M != L)
```
Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm]: this should be a proper faction check, maybe use the target list proc minus the attacked target?

File: code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm (hunk 1 above)
Comment (Tacolizard) [reply code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm]: it was like this before this PR

File: code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm (hunk 3 above)
Comment (Tacolizard) [reply code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm]: how do i do a 'proper faction check'

File: code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm (hunk 2 above)
Comment (MoreRobustThanYou) [reply code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm]: https://seaworld.org/animal-info/animal-infobooks/gorilla/diet-and-eating-habits/

**Comment by Armhulen** [comment]
y

**Comment by Tacolizard** [comment]
kinda requires a refactor of how humans are thrown in order to get what i want. the other features are relatively minor so i closed it.

**Pull request closed without merging**